src/components/Layout.js

```diff
@@ -24,19 +26,40 @@ const Layout = ({ children, pageTitle, fullPage }) => {
   )
 
   return (
-    <div
-      className={`${isRtl ? 'rtl' : ''} ${
-        language === 'am' ? 'amharic' : ''
-      } bg-gray-100 leading-normal tracking-normal`}
-    >
-      <Header />
+    <div>
+      <Helmet titleTemplate="%s | barmaga.io">
+        <meta property="og:type" content="website" />
+        <meta property="og:title" content={t('landingPage.heroText')} />
+        <meta
+          property="og:description"
+          content={t('landingPage.heroSubtitle')}
+        />
+        <meta property="og:url" content="https://barmaga.io" />
+        <meta
+          property="og:image"
+          content="https://cdn.nyaladev.com/barmaga.io/barmaga_logo_sm.svg"
```

Comment:
does twitter accept an svg image? I thought it has to be `png` or `jpeg`?

Reply:
I don't know